api-reference/docstubs/api/calendar-delete-events.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Delete events"
+description: "Delete an events object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: "**TODO: Add MS prod**"
+doc_type: apiPageType
+---
+
+# Delete events
+
+Namespace: microsoft.graph
+
+Delete an events object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+DELETE /invitations/{invitationsId}/invitedUser/messages/{messageId}/event/calendar/events
```

Comment:
@wandeg , @dkershaw10 line 31: huh?
When I looked at the filename, I was guessing somehow we generated a separate topic for deleting an event in a specified calendar, which BTW, the Ruby tool seems to have aggregated that under event-delete.md, and that's what we publish on the live site too. I checked some more recent generation using the Ruby tool, I think that's the case. 

However, the syntax in line 31 is not even that. In short, it looks like this is picking up the nav property invitedUser (which is of the user type), and expanding more nav properties - messages of user, then event of message, then...

Even if we think this is syntactically correct based on the CSDL, would we end up generating a lot more topics for authors to exclude?